common/core/web/keyboard-processor/src/text/keyboardProcessor.ts

```diff
@@ -448,25 +445,10 @@ namespace com.keyman.text {
           keyShiftState &= ~Codes.modifierCodes['RALT'];
         }
 
-        for(i=0; i < lockNames.length; i++) {
-          if(lockStates & Codes.stateBitmasks[lockNames[i]]) {
-            this.stateKeys[lockKeys[i]] = !!(lockStates & Codes.modifierCodes[lockNames[i]]);
-          }
-        }
-      } else if(d) {
-        keyShiftState |= v;
-
-        for(i=0; i < lockNames.length; i++) {
-          if(v & Codes.stateBitmasks[lockNames[i]]) {
-            this.stateKeys[lockKeys[i]] = true;
-          }
-        }
```

Comment:
Per my analysis, this code path was never called (would require calling with `_UpdateVKState(null, v, 1)`, which never happened)